server/README.md

```diff
@@ -1,23 +1,20 @@
 # Atum Web Service
 
-## How to build and run
+## How to build and run (will be updated soon)
```

Comment:
hmm, what do you mean will be updated soon?

Reply:
I mean the part about packaging for deployment. The first code block.